### PR TITLE
Add required parameter in shipping carrier validation rule

### PR DESCRIPTION
**Pull request by magesanjay**
## Scope of this PR

- __Technical__: Add Required parameter in shipping carrier validation rule

## Purpose of this PR

When this pull request is merged, it will add Required parameter in custom shipping carrier validation rule.

## Affected URLs

- https://devdocs.magento.com/guides/v2.1/howdoi/checkout/checkout_carrier.html
- https://devdocs.magento.com/guides/v2.2/howdoi/checkout/checkout_carrier.html
- https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_carrier.html

## Links to source code

Codebase Urls:

- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Fedex/view/frontend/web/js/model/shipping-rates-validation-rules.js#L12
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Fedex/view/frontend/web/js/model/shipping-rates-validation-rules.js#L12
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Fedex/view/frontend/web/js/model/shipping-rates-validation-rules.js#L13

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/howdoi/checkout/checkout_carrier.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ define(
                     },
                     'country_id': {
                         'required': true
+                    },
+                    'city': {
```
Comment (ronak2ram) [inline guides/v2.1/howdoi/checkout/checkout_carrier.md]: Hi @magesanjay 
Can you please add city field in Line no:39 Like  **Country**, **Zip Code** and **City**

Comment (magesanjay) [reply guides/v2.1/howdoi/checkout/checkout_carrier.md]: Hello @ronak2ram
I have updated content also as per your request.

Comment (ronak2ram) [reply guides/v2.1/howdoi/checkout/checkout_carrier.md]: @magesanjay thanks for the update PR. PR looks good now.

**Pull request merged**